### PR TITLE
Hts 477 update page titles

### PR DESCRIPTION
**Pull request by thebailers**
Update page titles, urls and headings

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    <input id="radio-inline-2" type="radio" name="radio-inline-group" value=false>
+                    <label for="radio-inline-2">
+                        @Messages("hts.cannot-change-email.opt2")
+                    </label>
+                </div>
+            </fieldset>
+
+        </div>
+
+        @submit('_buttonClass -> "", 'id → "email-error-continue"){ Continue }
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html]: The string `Continue` looks like it belongs in the messages file

File: conf/app.routes [hunk 2]
```diff
@@ -13,23 +13,27 @@ POST       /apply-for-help-to-save/apply                    @uk.gov.hmrc.helptos
 
 
 GET        /select-email                                    @uk.gov.hmrc.helptosavefrontend.controllers.RegisterController.getSelectEmailPage
-GET        /access-account                                  @uk.gov.hmrc.helptosavefrontend.controllers.AccessAccountController.accessAccount
+GET        /no-account                                      @uk.gov.hmrc.helptosavefrontend.controllers.AccessAccountController.accessAccount
```
Comment (AndyHWChung) [inline conf/app.routes]: we may want to keep this endpoint and create a new endpoint for` /no-account` - this endpoint is the one for when a user clicks `Access Your Account` - if they don't have an account they should redirect to `/no-account`

Comment (AndyHWChung) [reply conf/app.routes]: (I can sort this one out)

File: app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+@()(implicit htsContext: HtsContext, request: Request[_], messages: Messages)
+
+@uk.gov.hmrc.helptosavefrontend.views.html.main_template(
+title = messages("hts.cannot-change-email-try-later.page-title"),
+sidebarLinks = None,
+bodyClasses = None) {
+
+ <h1 class="heading-xlarge">@Messages("hts.cannot-change-email-try-later.title")</h1>
+
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html]: is this missing copy?

File: app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html (hunk 1 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html]: Yeah and line 49 in file select_email.scala.html is

                <input id="add-new-email" type="radio" name="email" value="No" @if(hasInvalidEmail){ checked }>

so checked should also be in messages?

File: app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html (hunk 3 above)
Comment (Jackie1988) [reply app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html]: how do you mean?

Comment (AndyHWChung) [reply app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email_try_later.scala.html]: I meant is this file missing the content  that was supposed to go in it - turns out it is but the addition of the actual content is part of another ticket

File: app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html (hunk 1 above)
Comment (AndyHWChung) [reply app/uk/gov/hmrc/helptosavefrontend/views/register/cannot_change_email.scala.html]: `checked` is not a string here - it's an html attribute

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**